packages/arb-token-bridge-ui/src/hooks/__tests__/useBalance.test.tsx

```diff
@@ -82,54 +87,8 @@ describe('useBalance', () => {
     expect(erc20Balances).toBeNull()
   })
 
-  it('getter return null for missing chainId', async () => {
```

Comment:
why is this test deleted?

Reply:
It became invalid after the changes in useBalance. We accept chainId, not provider anymore